plugins/08_BattleForAzeroth/zones/drustvar.lua

```diff
@@ -845,3 +845,68 @@ map.nodes[21602880] = Safari.ShackCrab({
         })
     }
 }) -- ShackCrab
+
+
+-------------------------------------------------------------------------------
+--------------------------- CURSED GAME HUNTER --------------------------------
+-------------------------------------------------------------------------------
+
+local Cursed = Class('Cursed', Collectible, {
+    icon = 1027879,
+    group = ns.groups.CURSED_HUNTER,
+    sublabel = L['cursed_hunter_note']
+})
+
+function Cursed.getters:label() return "Cursed animals" end
```

Comment:
`Cursed.getters:label()` could be replaced by adding `label = L['Cursed animals']` to the Class.
And `L['Cursed animals']` should be added to localizations.